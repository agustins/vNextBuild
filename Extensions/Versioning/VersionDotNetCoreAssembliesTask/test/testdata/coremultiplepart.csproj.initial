<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.1</TargetFramework>
    <AssemblyName>VersionedLibrary</AssemblyName>
    <AnotherVersion>1.2.3.4</AnotherVersion>
    <VersionAnother>1.2.3.4</VersionAnother>
    <AssemblyVersion>1.2.3.4</AssemblyVersion>
    <Version>1.2.3.4</Version>
    <Description>A description</Description>
  </PropertyGroup>

</Project>

