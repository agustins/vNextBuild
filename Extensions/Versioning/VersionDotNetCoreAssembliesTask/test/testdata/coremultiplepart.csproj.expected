<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.1</TargetFramework>
    <AssemblyName>VersionedLibrary</AssemblyName>
    <AnotherVersion>1.2.3.4</AnotherVersion>
    <VersionAnother>1.2.3.4</VersionAnother>
    <AssemblyVersion>9.9.9.9</AssemblyVersion>
    <Version>9.9.9.9</Version>
    <Description>A description</Description>
  </PropertyGroup>

</Project>

